build/crds/sedna.io_federatedlearningjobs.yaml

```diff
@@ -13338,6 +13348,25 @@ spec:
                   - template
                   type: object
                 type: array
+              transmitter:
+                description: Transmitter describes a transmitter
+                properties:
+                  s3:
+                    description: S3Transmitter
+                    properties:
+                      aggDataPath:
+                        type: string
+                      credentialName:
+                        type: string
+                    required:
+                    - aggDataPath
+                    - credentialName
```

Comment:
why is `credentialName` required? how about public aggDataPath?

Reply:
`aggDataPath` may be public